common/hexutil/json.go

```diff
@@ -187,6 +206,25 @@ func (b *Big) String() string {
 	return EncodeBig(b.ToInt())
 }
 
+// ImplementsGraphQLType returns true if Big implements the provided GraphQL type.
+func (b Big) ImplementsGraphQLType(name string) bool { return name == "BigInt" }
+
+// UnmarshalGraphQL unmarshals the provided GraphQL query data.
+func (b *Big) UnmarshalGraphQL(input interface{}) error {
+	var err error
+	switch input := input.(type) {
```

Comment:
prefer the syntax `input, ok := input.(type)` so that an error will be returned instead of panicking.

Reply:
I don't think that's possible here? This is a type switch, not a cast.